src/sql/src/statements/copy.rs

```diff
@@ -44,6 +52,52 @@ impl CopyTable {
         &self.format
     }
 }
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct CopyTableFrom {
```

Comment:
https://docs.rs/sqlparser/latest/sqlparser/ast/enum.Statement.html#variant.Copy

Seems COPY TO and COPY FROM use almost the same options, could combine these two structs? Like what sqlparser does. @v0y4g3r @fengjiachun

Reply:
> https://docs.rs/sqlparser/latest/sqlparser/ast/enum.Statement.html#variant.Copy
> 
> Seems COPY TO and COPY FROM use almost the same options, could combine these two structs? Like what sqlparser does. @v0y4g3r @fengjiachun

Sounds good, but I don't think it's related with this PR.

Reply:
Good suggestion, I think we can combine the two structures or leave a todo for now.